spec/unit/fixity_service_spec.rb

```diff
@@ -4,6 +4,32 @@
   let(:service) { described_class.new(uri) }
   let(:uri) { RDF::URI("http://path/to/resource") }
 
+  let(:passing_fedora44_response_body) {
+    <<-EOF
+@prefix premis: <http://www.loc.gov/premis/rdf/v1#> .
+
+<http://127.0.0.1:8080/rest/dev/0k/22/5b/04/0k225b04p/files/9f296a1f-10e7-44a3-83eb-4811d611edc6/fcr:versions/version1> premis:hasFixity <http://127.0.0.1:8080/rest/dev/0k/22/5b/04/0k225b04p/files/9f296a1f-10e7-44a3-83eb-4811d611edc6/fcr:versions/version1#fixity/1493843767961> .
+
+<http://127.0.0.1:8080/rest/dev/0k/22/5b/04/0k225b04p/files/9f296a1f-10e7-44a3-83eb-4811d611edc6/fcr:versions/version1#fixity/1493843767961> a premis:Fixity , premis:EventOutcomeDetail ;
+  premis:hasEventOutcome "SUCCESS"^^<http://www.w3.org/2001/XMLSchema#string> ;
+  premis:hasMessageDigest <urn:sha1:b995eef5262dd1c74f0ed9c96be1f404394d45dc> ;
+  premis:hasSize "103945"^^<http://www.w3.org/2001/XMLSchema#long> .
+  EOF
+  }
+
+  let(:failing_fedora44_response_body) {
+    <<-EOF
+@prefix premis: <http://www.loc.gov/premis/rdf/v1#> .
+
+<http://127.0.0.1:8080/rest/dev/ks/65/hc/20/ks65hc20t/files/e316b4b5-4627-44f8-9fdb-d2016e0e7380/fcr:versions/version3> premis:hasFixity <http://127.0.0.1:8080/rest/dev/ks/65/hc/20/ks65hc20t/files/e316b4b5-4627-44f8-9fdb-d2016e0e7380/fcr:versions/version3#fixity/1493844791463> .
+
+<http://127.0.0.1:8080/rest/dev/ks/65/hc/20/ks65hc20t/files/e316b4b5-4627-44f8-9fdb-d2016e0e7380/fcr:versions/version3#fixity/1493844791463> a premis:Fixity , premis:EventOutcomeDetail ;
+  premis:hasEventOutcome "BAD_CHECKSUM"^^<http://www.w3.org/2001/XMLSchema#string> , "BAD_SIZE"^^<http://www.w3.org/2001/XMLSchema#string> ;
+  premis:hasMessageDigest <urn:sha1:1a89571e25dd372563a10740a883e93f8af2d146> ;
+  premis:hasSize "1878582"^^<http://www.w3.org/2001/XMLSchema#long> .
+EOF
+  }
+
```

Comment:
Are you thinking of a fixture file that just has the text in it, that just gets read in with `File.read`? Personally, I like having the data right here in the file so a developer can look at it and see exactly what is being tested. I think especially important for a mock of an external serivce I think, where it can easily get out of sync with what the service is actually returning (as I think it had), and it's good for it to be front and center for any developer looking at it. 

But it's easy enough to make a fixture file if the consensus is a different opinion?